stable/contour/templates/service.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "contour.service.fullname" . }}
```

Comment:
had to switch to `name: {{ template "contour.fullname" . }}` to have a fully qualified name include release name

Reply:
Added the fix